db.go

```diff
@@ -52,11 +52,44 @@ func (net *Network) GetName() string {
 	return net.Addr
 }
 
+type MessageFilter int
+
+const (
+	FilterNone      MessageFilter = 0
```

Comment:
I wonder if we should have a `FilterDefault` value to mean "user didn't specify anything". This could be used in the future to set a per-user preference instead of having to set the preference on each channel.

This would increase complexity a little bit.